automation/src/test/scala/org/broadinstitute/dsde/firecloud/component/Tags.scala

```diff
@@ -0,0 +1,15 @@
+package org.broadinstitute.dsde.firecloud.component
+
+import org.openqa.selenium.{By, WebDriver}
+
+case class Tags (queryString: QueryString)(implicit webDriver: WebDriver) extends Component(queryString) {
+
+  def getTags: List[String] = {
+    import scala.collection.JavaConversions._
+    val elm = find(query)
+    if (elm.isDefined)
+      elm.get.underlying.findElements(By.tagName("div")).toList.map(_.getText)
```

Comment:
hunting for divs feels a little fragile, in case we ever change the layout. Alternative would be to update `render-tags`/`render-tag` to include a class or data-test-id or the like. I'm fine either way for this PR, your call.

Reply:
There is code to do exactly that in my PR. Relevant part
```
([tag] (render-tag {:data-test-id (str (string/lower-case tag) "-tag")} tag))
```

Reply:
chose a different solution - using the title